tests/python_package_test/test_engine.py

```diff
@@ -1044,15 +1045,21 @@ def test_contribs_sparse():
     # convert data to dense and get back same contribs
     contribs_dense = gbm.predict(X_test.toarray(), pred_contrib=True)
     # validate the values are the same
-    np.testing.assert_allclose(contribs_csr.toarray(), contribs_dense)
+    if platform.machine() == 'aarch64':
+        np.testing.assert_allclose(contribs_csr.toarray(), contribs_dense, rtol=1, atol=1e-12)
+    else:
+        np.testing.assert_allclose(contribs_csr.toarray(), contribs_dense)
```

Comment:
**Sometimes** `contribs_sparse` tests are failing, so need to relax assertion tolerance
```
>       np.testing.assert_allclose(contribs_csr_arr_re, contribs_dense)
E       AssertionError: 
E       Not equal to tolerance rtol=1e-07, atol=0
E       
E       Mismatched elements: 20 / 840 (2.38%)
E       Max absolute difference: 2.22044605e-16
E       Max relative difference: 1.
E        x: array([[ 0.000000e+00,  0.000000e+00,  0.000000e+00,  0.000000e+00,
E               -3.840704e-02, -1.261461e-01, -3.519505e-02,  1.271551e-04,
E                0.000000e+00, -2.690382e-04, -1.218832e-01,  1.221108e-04,...
E        y: array([[ 0.000000e+00,  0.000000e+00, -4.847735e-17,  0.000000e+00,
E               -3.840704e-02, -1.261461e-01, -3.519505e-02,  1.271551e-04,
E                0.000000e+00, -2.690382e-04, -1.218832e-01,  1.221108e-04,...
```